From a31381771678d1dacd5b66166fab3412d742078e Mon Sep 17 00:00:00 2001
From: Chenyan Feng <ella.feng@nxp.com>
Date: Mon, 3 Jul 2017 11:32:18 +0800
Subject: [PATCH 2032/5242] MGS-3054 dts: update gpu shader clock to meet
 design

commit  641b4400f8aedc02f2d5dbfc31d9047ca32a8142 from
https://source.codeaurora.org/external/imx/linux-imx.git

changed gpu shader clock from 1G to 800Mhz per design
also updated gpu shader parent from SYS2_PLL to SYS1_PLL.
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index bbda606..55bc65c 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -582,8 +582,8 @@
 		clocks = <&clk IMX8MQ_CLK_GPU_ROOT>, <&clk IMX8MQ_CLK_GPU_SHADER_DIV>, <&clk IMX8MQ_CLK_GPU_AXI_DIV>;
 		clock-names = "gpu3d_clk", "gpu3d_shader_clk", "gpu3d_axi_clk";
 		assigned-clocks = <&clk IMX8MQ_CLK_GPU_CORE_SRC>, <&clk IMX8MQ_CLK_GPU_SHADER_SRC>, <&clk IMX8MQ_CLK_GPU_AXI_SRC>, <&clk IMX8MQ_CLK_GPU_AHB_SRC>;
-		assigned-clock-parents = <&clk IMX8MQ_SYS1_PLL_800M>, <&clk IMX8MQ_SYS2_PLL_1000M>, <&clk IMX8MQ_SYS1_PLL_800M>, <&clk IMX8MQ_SYS1_PLL_800M>;
-		assigned-clock-rates = <800000000>, <1000000000>, <800000000>, <800000000>;
+		assigned-clock-parents = <&clk IMX8MQ_SYS1_PLL_800M>, <&clk IMX8MQ_SYS1_PLL_800M>, <&clk IMX8MQ_SYS1_PLL_800M>, <&clk IMX8MQ_SYS1_PLL_800M>;
+		assigned-clock-rates = <800000000>, <800000000>, <800000000>, <800000000>;
 		status = "disabled";
 	};
 
-- 
1.7.9.5

